packages/moonstone/Spinner/Spinner.module.less

```diff
@@ -4,150 +4,99 @@
-	/* Small bounce forward */
-	38%       { -webkit-transform: rotateZ(360 + @moon-spinner-ball-angle + 10); }
-	45%, 66%  { -webkit-transform: rotateZ(360 + @moon-spinner-ball-angle); }
-
-	66.01%    { -webkit-transform: rotateZ(@moon-spinner-ball-angle); }
-	/* Slow down toward the top */
-	81%       { -webkit-transform: rotateZ(160deg); }
-	91%       { -webkit-transform: rotateZ(200deg); }
-	97%       { -webkit-transform: rotateZ(225deg); }
-	/* Speed back up as the ball falls */
+@keyframes rotate2 {
+	0%, 100% { transform: rotate(0turn); }
+	33% { transform: rotate((2turn / 8)); }
 }
-@keyframes spinBall {
-	0%        { transform: rotateZ(@moon-spinner-ball-angle); }
-	/* Slow down toward the top */
-	15%       { transform: rotateZ(160deg); }
-	25%       { transform: rotateZ(200deg); }
-	30%       { transform: rotateZ(225deg); }
-	/* Speed back up as the ball falls */
-	/* Impact at last ball */
-	33%       { transform: rotateZ(360 - (@moon-spinner-ball-angle * 3)); }
-	39%, 67%  { transform: rotateZ(360 - @moon-spinner-ball-angle); }
-	/* Small bounce forward */
-	72%       { transform: rotateZ(360 + @moon-spinner-ball-angle + 10); }
-	79%, 100% { transform: rotateZ(360 + @moon-spinner-ball-angle); }
+@keyframes rotate3 {
+	0%, 100% { transform: rotate(0turn); }
+	33% { transform: rotate((3turn / 8)); }
 }
-@keyframes spinBall2 {
-	/* Small bounce forward */
-	0%        { transform: rotateZ(360 - @moon-spinner-ball-angle); }
-	5%        { transform: rotateZ(360 + @moon-spinner-ball-angle + 10); }
-	12%, 33%  { transform: rotateZ(360 + @moon-spinner-ball-angle); }
-
-	33.01%    { transform: rotateZ(@moon-spinner-ball-angle); }
-	/* Slow down toward the top */
-	48%       { transform: rotateZ(160deg); }
-	58%       { transform: rotateZ(200deg); }
-	63%       { transform: rotateZ(225deg); }
-	/* Speed back up as the ball falls */
-	/* Impact at last ball */
-	67%       { transform: rotateZ(360 - (@moon-spinner-ball-angle * 3)); }
-	72%, 100% { transform: rotateZ(360 - @moon-spinner-ball-angle); }
-}
-@keyframes spinBall3 {
-	/* Impact at last ball */
-	0%, 100%  { transform: rotateZ(360 - (@moon-spinner-ball-angle * 3)); }
-	5%, 33%   { transform: rotateZ(360 - @moon-spinner-ball-angle); }
-	/* Small bounce forward */
-	38%       { transform: rotateZ(360 + @moon-spinner-ball-angle + 10); }
-	45%, 66%  { transform: rotateZ(360 + @moon-spinner-ball-angle); }
-
-	66.01%    { transform: rotateZ(@moon-spinner-ball-angle); }
-	/* Slow down toward the top */
-	81%       { transform: rotateZ(160deg); }
-	91%       { transform: rotateZ(200deg); }
-	97%       { transform: rotateZ(225deg); }
-	/* Speed back up as the ball falls */
+@keyframes rotate4 {
+	0%, 100% { transform: rotate(0turn); }
+	33% { transform: rotate((4turn / 8)); }
 }
 
 .spinner {
 	line-height: @moon-body-line-height;
 
-	.ballDecorator {
-		border-radius: (@moon-spinner-size / 2);
+	.bg {
 		position: relative;
+		border-radius: 999px;
```

Comment:
Large enough. This can't simply be a percent if you don't want an oval (when the height and width don't match) and using a calculation could potentially get out of sync if your calculation isn't spot-on, always.